fixtures/tracks/animal/exercises/TRACK_HINTS.md

```diff
@@ -0,0 +1 @@
+This is the content of the track hints file
```

Comment:
Ahhhh, I completely missed this.

> The question is whether it should support tracks that have an exercises directory, but not all exercises are in it.

No, it's one or the other. Either the track has moved its exercises or it hasn't.